README.md

```diff
@@ -287,6 +287,7 @@ var server = ParseServer({
     // 2. a callback function to be invoked to validate the password  
     validatorCallback: (password) => { return validatePassword(password) }, 
     doNotAllowUsername: true, // optional setting to disallow username in passwords
+    daysBeforeExpiry: 90, // threshold (days) for password expiry. Login fails if user does not reset the password within this period after signup/last reset. 
```

Comment:
@bhaskaryasa 
Can you pls change `daysBeforeExpiry` to 
`maximumPasswordAge`? Pls refer to https://technet.microsoft.com/en-us/library/ff741764.aspx for good naming conventions for these variable names.